lib/src/components/joystick/joystick_directional.dart

```diff
@@ -151,17 +151,15 @@ class JoystickDirectional extends BaseComponent with Draggable, HasGameRef {
         radAngle: radAngle,
       ));
     } else {
-      if (_knobRect != null) {
-        final diff = _dragPosition - _knobRect.center.toVector2();
-        _knobRect = _knobRect.shift(diff.toOffset());
-      }
+      final diff = _dragPosition - _knobRect.center.toVector2();
+      _knobRect = _knobRect.shift(diff.toOffset());
     }
   }
 
   @override
   bool containsPoint(Vector2 point) {
-    final directional = _backgroundRect?.inflate(50.0);
-    return directional?.containsPoint(point) == true;
+    final directional = _backgroundRect.inflate(50.0);
+    return directional.containsPoint(point) == true;
```

Comment:
I think we can just return the `containsPoint` method result now. Before the comparison made sense because it could be null.